src/models/stores/user.ts

```diff
@@ -5,18 +5,14 @@ const initials = require("initials");
 export const UserTypeEnum = types.enumeration("type", ["student", "teacher"]);
 export type UserType = typeof UserTypeEnum.Type;
 
-export const PortalClass = types.model("PortalClass", {
+export const ClueClassOffering = types.model("ClueClassOffering", {
```

Comment:
User is a cross-application model, so it shouldn't contain application-specific code. Is there a portal-generic name for what's being modeled here like `PortalClassOffering`?

Reply:
Done.